db/migrate/20201224095619_remove_ovirt_use_api_from_attr.rb

```diff
@@ -0,0 +1,13 @@
+class RemoveOvirtUseApiFromAttr < ActiveRecord::Migration[6.0]
+  def change
+    compute_resources = ComputeResource.all.unscoped.where(type: 'Foreman::Model::Ovirt')
```

Comment:
not important but you could also do:
```suggestion
    compute_resources = Foreman::Model::Ovirt.unscoped
```